juniper_official/Interfaces/interface-throughput.playbook

```diff
@@ -0,0 +1,14 @@
+/*
+ * Playbook contains multiple rules which monitor interfaces bandwidth 
+ * utilization and notifies when anomalies are found.
+ * 
+ * 1) Rule check-interface-bandwidth-netconf,, detects the interface 
+ *    bandwidth and notifies anomalies when utilization is more than thresholds
+ */
```

Comment:
Modify as below
* 1) Rule check-interface-bandwidth-netconf, Notifies anomalies when interface throughput exceeds threshold.

Reply:
updated the rule comments